src/misc/common_macros.hpp

```diff
@@ -15,8 +15,13 @@
 #ifndef QASM_SIMULATOR_COMMON_MACROS_HPP
 #define QASM_SIMULATOR_COMMON_MACROS_HPP
 
-#if defined(__GNUC__) && defined(__x86_64__)
- #define GNUC_AVX2
+#if defined(_MSC_VER)
+ #include <intrin.h>
+#elif defined(__GNUC__) && defined(__x86_64__)
+  #ifndef __PPC64__
+     #define GNUC_AVX2
+     #include <cpuid.h>
+  #endif
```

Comment:
The inclusion header for AVX2 has been refactored in PR #1017 (already merged to master). So when you merge master to your branch, accept master changes regarding all this AVX2 specific things. All those includes have been moved to the file that actual needs them.

Reply:
Okay, great. Have merged that.